llvm/test/DebugInfo/Generic/assignment-tracking/instcombine/alloca-bitcast.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: opt -passes=instcombine -S %s -o - -experimental-assignment-tracking \
+; RUN: opt -passes=instcombine -S %s -o - \
```

Comment:
This doesn't seem to be related to DPC++ compiler. Why do we need this change?
Could you add a comment to the code?

Reply:
I'm assuming there was a wrong conflict resolution, so I just brought it up to date with the llorg version.